Scripts/Mac/BuildInstaller.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+CONFIGURATION=$1
+if [ -z $CONFIGURATION ]; then
+  CONFIGURATION=Debug
+fi
+
+PACKAGEVERSION=$2
+if [ -z $PACKAGEVERSION ]; then
+  PACKAGEVERSION="1.0.0.1"
+fi
+
+SCRIPTDIR="$(dirname ${BASH_SOURCE[0]})"
+
+# convert to an absolute path because it is required by `dotnet publish`
+pushd $SCRIPTDIR
+SCRIPTDIR="$(pwd)"
+popd
+
+SRCDIR=$SCRIPTDIR/../..
+ROOTDIR=$SRCDIR/..
+BUILDOUTPUT=$ROOTDIR/BuildOutput
+PUBLISHDIR=$ROOTDIR/Publish
+INSTALLROOT=$ROOTDIR/InstallRoot
+VFSFORGITDESTINATION="usr/local/vfsforgit"
+INSTALLERPACKAGENAME="VFSForGit.$PACKAGEVERSION"
+
+echo "SRCDIR: ${SRCDIR}"
+echo "ROOTDIR: ${ROOTDIR}"
+echo "BUILDOUTPUT: ${BUILDOUTPUT}"
+echo "PUBLISHDIR: ${PUBLISHDIR}"
+
+function CheckBuildIsAvailable()
+{
+	if [ ! -d $BUILDOUTPUT ] || [ ! -d $PUBLISHDIR ]; then
+  		echo "Could not find VFSForGit Build to package."
+  		exit 1
+	fi
+}
+
+function SetPermissions()
+{
```

Comment:
Same thing as CreateInstallerRoot. 

Additionally, I think if a script in pkgbuild creates the directory, the permissions shouldn't need to be changed (not sure if this prototype requires elevation. but if the install directory is owned by root, we might need more changes here)